src/serializer/ResidualHeapSerializer.js

```diff
@@ -695,6 +701,120 @@ export class ResidualHeapSerializer {
     return t.arrayExpression(initProperties);
   }
 
+  _getPropertyValue(properties: Map<string, any>, key: string) {
+    if (properties.has(key) === true) {
+      let val = properties.get(key);
+
+      if (val !== undefined) {
+        let descriptor = val.descriptor;
+
+        if (descriptor !== undefined) {
+          let descriptorValue = descriptor.value;
+
```

Comment:
What if the descriptor is an accessor property? I suspect you'd want to log an error in those cases (not supported).